apps/docs/app/features/routes/ressurser/kom-i-gang/Component.tsx

```diff
@@ -0,0 +1,45 @@
+import {
+  Heading,
+  Flex,
+  Image,
+  SimpleGrid,
+  Stack,
+  Text,
+} from "@vygruppen/spor-react";
+
+export default function Component() {
+  return (
+    <SimpleGrid columns={{ base: 1, md: 2 }} spacing={8}>
+      <Flex>
+        <Image
+          src="/images/component-examples/component-example-1.png"
+          alt="Componentexample"
+          rounded={"md"}
```

Comment:
Pirk: Her kan du droppe { og }, siden det bare er en vanlig tekststreng.
```suggestion
          rounded="md"
```